tutorials/07_conditional_distributions.ipynb

```diff
@@ -16339,6 +16339,229 @@
+   "metadata": {},
+   "source": [
+    "## Sampling conditional distributions\n",
+    "\n",
+    "So far, we have demonstrated how one can plot 2D conditional distributions with `conditional_pairplot()` and how one can compute the pairwise conditional correlation coefficient with `conditional_corrcoeff()`. In some cases, it can be useful to keep a subset of parameters fixed and to vary **more than two** parameters. This can be done by sampling the conditonal posterior $p(\\theta_i | \\theta_{j \\neq i}, x_o)$. As of `sbi` `v0.18.0`, this functionality requires using the [sampler interface](https://www.mackelab.org/sbi/tutorial/11_sampler_interface/). In this tutorial, we demonstrate this functionality on a linear gaussian simulator with four parameters. We would like to fix the forth parameter to $\\theta_4=0.2$ and sample the first three parameters given that value, i.e. we want to sample $p(\\theta_1, \\theta_2, \\theta_3 | \\theta_4 = 0.2, x_o)$. For an application in neuroscience, see [Deistler, Gonçalves, Macke, 2021](https://www.biorxiv.org/content/10.1101/2021.07.30.454484v4.abstract)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we will use SNPE, but the same also works for SNLE and SNRE. First, we define the prior and the simulator and train the deep neural density estimator:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "4683b28ba0614e87b33854d207e30da2",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Running 1000 simulations.:   0%|          | 0/1000 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      " Neural network successfully converged after 147 epochs."
+     ]
+    }
+   ],
+   "source": [
+    "from sbi.inference import SNPE, prepare_for_sbi, simulate_for_sbi\n",
+    "from sbi.inference import posterior_estimator_based_potential, MCMCPosterior\n",
+    "from sbi.utils import BoxUniform\n",
+    "import torch\n",
+    "\n",
+    "num_dim = 4\n",
+    "prior = BoxUniform(low=-2*torch.ones(num_dim), high=2*torch.ones(num_dim))\n",
+    "\n",
+    "def linear_gaussian(theta):\n",
+    "    return theta + 1.0 + torch.randn_like(theta) * 0.1\n",
+    "\n",
+    "simulator, prior = prepare_for_sbi(linear_gaussian, prior)\n",
+    "\n",
+    "inference = SNPE()\n",
+    "theta, x = simulate_for_sbi(simulator, prior, 1000)\n",
+    "posterior_estimator = inference.append_simulations(theta, x).train()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we follow the sampler interface and create a potential_function. The observation in this example is `x_o=[1, 1, 1, 1]`."
```

Comment:
Just watching this from the sidelines. the meaning of `potential_function` is not discussed here. I also checked that in none of the previous tutorials, a `potential_function` is mentioned. It might be worthwhile to share here, what mathematical meaning you connect with the `potential_function`. It gives readers some context.

Reply:
Thanks for the input Peter! The potential function is expained in the sampler interface [tutorial](https://www.mackelab.org/sbi/tutorial/11_sampler_interface/). Quote:
```
Next, we obtain the potential function. A potential function is a function of the parameter f(theta). The posterior is proportional to the product of likelihood and prior: p(theta|x) \propto p(x|theta)p(theta). The potential function is the logarithm of the right-hand side of this equation: f(theta) = log(p(x|theta)p(theta))
)
```

Reply:
I agree, I specifically added a pointer to it.